cluster/ci/monitoring/build_cop.json

```diff
@@ -871,6 +871,42 @@
             "type": "query",
             "useTags": false
          },
+         {
```

Comment:
This is suboptimal since you can only add a template variable to the entire dashboard from what I can tell. not to a specific plot. Oh well